src/index.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable linebreak-style */
+
+import './style.css';
+
+const todolist = [{
+  id: '',
+  todo: 'mmmmmmmmmmmm',
+  todostatus: '',
+}];
+
+const addtodo = (inputtodo) => {
+  if (inputtodo.length < 1) {
+    const message = document.getElementById('message');
+    message.innerHTML = 'Please, type a "to do" activity';
+  } else {
+    const message = document.getElementById('message');
+    message.innerHTML = '';
+    todolist.push(
+      {
+        id: todolist.length > 0 ? todolist[todolist.length - 1].id + 1 : 1,
+        todo: inputtodo,
+        todostatus: true,
+      },
+    );
+    localStorage.setItem('todolist', JSON.stringify(todolist));
+  }
+};
+
+const removetodo = (i) => {
+  console.log(todolist[i].todo);
+};
+
+const showtodolist = (todolist) => {
+  const totallist = document.getElementById('totallist');
+  totallist.innerHTML = '';
+  for (let i = 1; i < todolist.length; i += 1) {
```

Comment:
Hi @ShoiraTa 

Thank you for taking the time to review my project and for the advice provided.

Regarding the required changes, I have the following comments:

Change 1:  "Please set i < todolist.length; => i <= todolist.length;,
because if to do list has 1 item, it will not show since the condition will be false"
  
 What you point out as a possible error in my code, is in fact an intended behaviour, as in my code design I plan to leave allways the first position of the array empty, so when starting the loop I will not be showing that empty element. If you populate the array with test cases, you will see that it works perfectly.

In addition to this, the activity I'm asking for approval, is the first activity of the To Do List project (Project 1), which only require  implementation of the structure, not the functionalities. You can populate the array with test cases and see that the structure displayed is the required.

By the way, this is the description shown about the whole To Do List project in the instruction  https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/sneak_peek.md#:~:text=Project%201%3A%20List,and%20remove%20items. :

Project 1: List structure.
Project 2: Interactive list.
Project 3: Add and remove items.


Change 2: "Please make sure that on page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the index values for each task. But currently this function is not being called on page load and the static array is not being rendered."

The static array is being rendered, but is empty, becouse the saving function hasn't being implemented yet.

In this case, I think that the functionality mentioned belongs also to the second phase of the project, not the structure phase. Of course, I'will implements it in the second phase and the saved information will be displayed on page load.

Change 3: " Kindly delete unnecessary headings and add Getting Started instructions, because some people may not know how to get started with webpack application"

In previous reviews of other projects, I have erased non-used headings and reviewers have asked to restore them. They say that even if it is not used, a README template must be respected and the field should say "Not Available".

I see that it is a point that can be discussed and that Microverse, sadly, hasn't emitted a clear directive regarding this point.

Regarding this situation, I consider that adding "Not Available" to the empty fields should be enough.

And of course, I'll add the Getting Start instruction.

Finally, based on all the above considerations, I think this project can be approved as it is. And all asked changes will be done in the second phase.

Thank you a lot in advance.

Reply:
- [ ] Hi @DarioAlessioR , thanks for not hesitating to write your points.
Regarding your comments:
1. Currently you have an array of 1 item and when you iterate over it, the condition is false and that single item is not `seen`. I did not understand your plan to leave always the first position of the array empty. Why would you do that? to have index start at from 1 not 0? I can only recommend not to do that, it would not be the best approach. But of course you can experiment with it 😉 
2. On page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the index values for each task => This is the requirement by microverse, I only follow the guidelines, to get approved for this milestone you have to make the list of items appear on the page: 

<img width="1104" alt="Screenshot 2022-06-10 at 21 42 14" src="https://user-images.githubusercontent.com/77038610/173121861-08ac7c64-bd36-427b-afc4-b6410b6664a6.png">

 3. You can add "Not Available"  on empty headings. When it comes to readme, your task is to make sure that it looks clear, professional and describes the project properly, readme is like the face of your project the first thing people, recruiters will see. If you think empty sections look professional, then I have no objections.

Reply:
Hi @ShoiraTa .

Thank you again for your comments.

I will try to advance with what you are asking, just to try to complete the activity, and even when I strongly disagree with the extent of your requirement, which I continue to think belongs to the second phase of the whole project.

I have compared my project against many others students' projects that were approved with a lot less of what my project has. And this is not my opinion, but the other student's opinion.

I consider that your demands are unjustifiably blocking my advance in the program.

Thank you in advance.

Reply:
Please request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)  🙏🏻